assets/js/filter_trade_form.js

```diff
@@ -0,0 +1,51 @@
+const losingTeamSelects = document.querySelectorAll(
+  '.losing-team'
+)
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)